rb/TESTING.md

```diff
@@ -1,13 +1,24 @@
 # Ruby Testing Guide
 
-This guide helps contributors write tests in the Selenium Ruby codebase.
+This guide helps contributors write tests, maintain code style, and generate documentation for the Selenium Ruby bindings.
+
+## Local Development Setup
+
+Before running tests, navigate to the `rb/` directory and install the required dependencies.
```

Comment:
local development still requires running bazel to generate the atoms and devtools code